lib/evss/auth_headers.rb

```diff
@@ -49,6 +49,11 @@ def eauth_json
       }.to_json
     end
 
+    def iso8601(date)
+      return nil unless date
```

Comment:
would it be too heavy of a lift to write a failing test for this case?

Reply:
Seconding this